src/date-range-picker/index.tsx

```diff
@@ -112,7 +112,7 @@ const DateRangePicker = React.forwardRef(
       locale = '',
       startOfWeek,
       isDateEnabled = () => true,
-      dateDisabledReason = () => '',
+      dateDisabledReason,
```

Comment:
Pushing the default value further down so I can properly assess if a function was provided from the outside or not